### ISSUE TITLE
False error 'not a git repository' on docker build when app is in subdir or .git is in .dockerignore

### ISSUE THREAD

**Issue by mkungla** [gobuffalo/buffalo#2048]
### Description

```
RUN buffalo build --static -o /bin/app
fatal: not a git repository (or any of the parent directories): .git
```

but build succeeds.
 
### Steps to Reproduce the Problem

  Please describe in painful detail what you did (so others can play along with you) to get to this point. This includes things like the exact command(s) you used, or the `curl` command you used, that sort of thing.

  1. app in subdir  and use `COPY ./app /app`

### Expected Behavior

**no errors**

### Actual Behavior

`fatal: not a git repository (or any of the parent directories): .git`

### Info

Please run `buffalo info` and paste the information below where it says "PASTE_HERE".

```
Buffalo version is: v0.16.14 
```

**Comment by leoromanovsky** [comment]
Related to: https://github.com/gobuffalo/buffalo/issues/2028

I fixed the issue locally by inserting: 

```
RUN git init
```

Into my Dockerfile; will submit a PR.

**Comment by kathanparikh** [comment]
Hi

**Comment by paganotoni** [comment]
This one is solved by v0.16.16
